nvflare/fuel/f3/streaming/byte_receiver.py

```diff
@@ -41,115 +41,226 @@
 ACK_INTERVAL = 1024 * 1024 * 4
 READ_TIMEOUT = 300
 COUNTER_NAME_RECEIVED = "received"
+
+# Read result status
 RESULT_DATA = 0
-RESULT_WAIT = 1
+RESULT_NO_DATA = 1
 RESULT_EOS = 2
 
 
 class RxTask:
     """Receiving task for ByteStream"""
 
-    def __init__(self, sid: int, origin: str):
+    rx_task_map = {}
+    map_lock = threading.Lock()
+    timeout = CommConfigurator().get_streaming_read_timeout(READ_TIMEOUT)
+    ack_interval = CommConfigurator().get_streaming_ack_interval(ACK_INTERVAL)
+    max_out_seq = CommConfigurator().get_streaming_max_out_seq_chunks(MAX_OUT_SEQ_CHUNKS)
```

Comment:
This requires that the CommConfigurator must be fully working at the time that the RxTask class is initialized. Not sure this can be met.

Reply:
Moved to constructor.